macros/blueprint-proc-macro-core/src/lib.rs

```diff
@@ -27,6 +27,8 @@ pub enum FieldType {
     Int64,
     /// A field of `u128` type.
     Uint128,
+    /// A field of `u256` type
+    U256,
```

Comment:
```suggestion
    Uint256,
```
Or is this something special that shouldn't follow the other variants?

Reply:
Special case